src/reporting/ReportingWindow.sol

```diff
@@ -44,9 +46,9 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable {
         return true;
     }
 
-    function createNewMarket(uint256 _endTime, uint256 _numOutcomes, uint256 _payoutDenominator, int256 _feePerEthInWei, address _denominationToken, address _creator, int256 _minDisplayPrice, int256 _maxDisplayPrice, address _automatedReporterAddress, int256 _topic) public payable returns (IMarket _newMarket) {
-        require(MINIMUM_OUTCOMES <= _numOutcomes && _numOutcomes <= MAXIMUM_OUTCOMES);
-        require(MINIMUM_PAYOUT_DENOMINATOR <= _payoutDenominator);
```

Comment:
Should these constants be removed from this contract since it looks like they aren't being used anymore?